src/plugins/intel_cpu/src/node.cpp

```diff
@@ -1303,11 +1299,6 @@ Node* Node::NodesFactory::create(const std::shared_ptr<ov::Node>& op, const Grap
     };
     Node *newNode = nullptr;
     std::string errorMessage;
-    {
-        std::unique_ptr<Node> ol(createNodeIfRegistered(intel_cpu, Type::Generic, op, context));
-        if (ol != nullptr && ol->created(context->getExtensionManager()))
-            newNode = ol.release();
-    }
```

Comment:
As we discussed, we have to keep this functionality for a while.

Reply:
@maxnick you have said that ov::intel_cpu::node::Generic was legacy  and we can remove this node, right? Now, we still  keep old extension until 24.0, and can first remove this node? It was involved in more legacy data classes that also need cleanup.

Reply:
ov::intel_cpu::node::Generic is a legacy node for sure, but it's required to support the old extension mechanism `ie_iextension.h`, which we may not remove until the legacy API is removed. As long as the legacy API is a part of the product, we have to support the legacy extension mechanism as well.

Reply:
ok, let's keep it 23.3.
I have created a ticket to remove it in 24.0: https://jira.devtools.intel.com/browse/CVS-125247